src/ui/views/input.coffee

```diff
@@ -243,7 +243,12 @@ maybeFocus = ->
         el.focus() if el
 
 preparemessage = (ev) ->
-    if models.viewstate.convertEmoji
+    if models.viewstate.commandsEnabled
+        element = document.getElementById "message-input"
+        # we want to match / and ignore //
+        if element.value.substr(0,1) == "/" and element.value.substr(1,1) != "/"
+            element.value = performCommand(models, element.value)
+    else if models.viewstate.convertEmoji
```

Comment:
breaks current functionality of emojis, as writing `:)` and enter does not convert emoji before sending.

just remove the else please :)